NETPROGRAM/String_Exercise/Exercise01.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.String_Exercise
+{
+    public class Exercise01
+    {
+
+        public string SnakeCase(string s)
+        {
+            var snakeString = s.ToLower();
+            for(var i = s.Length - 2 ; i > 0 ; i--)
+            {
+
+                if (s.ToLower()[i] - s[i] > 0)
+                {
+                    snakeString = snakeString.Insert(i, "_");
```

Comment:
Nếu có nhiều hơn 1 (ví dụ 4 5) chổ cần điền dấu `_` thì sao?

Reply:
Thuật toán vẫn đảm bảo được mà em.

Reply:
Vẫn dùng bt mà em.